src/util/colors.ts

```diff
@@ -9,4 +9,4 @@ export const LIGHT_GRAY = '#F0EEEE';
 export const LIGHT_GRAY_DISABLED = '#C4C4C4';
 export const DARK_GRAY = 'rgb(60, 60, 60)';
 export const DARK_GRAY_TRANSPARENT = 'rgba(60, 60, 60, .5)';
-export const RED = '#E10000';
+export const RED = '#E10000';
```

Comment:
This file is out of scope of the card. Is this change a remnant of the commit where you updated colors yourself before you merged to master?

Reply:
Yes. Every time I pulled from the upstream, my forked version of master would update to the correct colors.ts file. But when  I rebased my branched with my master, the old colors.ts file in my branch would overwrite the up to date color.ts file in master.

So, I had to copy and paste the correct version of colors.ts into my branch.

 I also thought I had to submit a new pull request once I finished the changes on this.

That PR is #36. How do we resolve that?

Reply:
Alright! We can go through it over the vid chat :)

And you're right about the new pull request, I'll close this one now!